.gitignore

```diff
@@ -13,3 +13,7 @@ cache
 ivy2.tgz
 ivy-tmp
 proxy-recorder/user-files
+
+# Bundler
+.bundle
+vendor
```

Comment:
I noticed that beaker will generate a `log` directory - might as well add that too